packages/tupaia-web/src/views/LandingPage/MultiProjectLandingPage.tsx

```diff
@@ -74,6 +75,9 @@ export function MultiProjectLandingPage({
   projects,
   includeNameInHeader,
 }: MultiProjectLandingPageProps) {
+  const { landingPageUrlSegment } = useParams();
+  const { isLandingPage } = useLandingPage(landingPageUrlSegment);
```

Comment:
You don't really need this check in here, because in this view it will always be a landing page, and on the projects modal it will never be a landing page